tests/scientific/test_simulation.py

```diff
@@ -121,3 +122,41 @@ def test_v5_gap_junction():
     assert spikes[1].size() == 1
     assert spikes[1][0] == 75779
     assert spikes[0][0] == pytest.approx(23.075)
+
+
+def test_v5_gap_junction_corrections(capsys):
+    import json
+    from neurodamus import Neurodamus
+    from neurodamus.core.configuration import SimConfig
+
+    # Add beta_features section for gj user corrections
+    config_file = str(SIM_DIR / "v5_gapjunctions" / "simulation_config.json")
+    with open(config_file) as f:
+        sim_config_data = json.load(f)
+    sim_config_data["output"]["output_dir"] = "$CURRENT_DIR/output_gj_corrections"
+    sim_config_data["beta_features"] = {
+        "gapjunction_target_population": "default",
+        "determanisitc_stoch": True,
+        "procedure_type": "validation_sim",
+        "gjc": 0.2,
+        "load_g_pas_file": "/gpfs/bbp.cscs.ch/project/proj55/amsalem/gap_junctions/circ19_11_2019_gjs_19_20_20/rm_correction/Circ_mc2_Rt_Remove_ch_all_Det_stoch_True_Dis_holding_False_Correc_type_impedance_tool_Cm0p01_Num_iter_10/num_0/g_pas_passive.hdf5", # noqa
+        "manual_MEComboInfo_file": "/gpfs/bbp.cscs.ch/project/proj55/amsalem/gap_junctions/circ19_11_2019_gjs_19_20_20_new_holding/find_holding_current/vc_ampFile/Circ_mc2_Rt_Remove_ch_none_Det_stoch_True_Dis_holding_False_gjc0p2_Change_mecomb_False_manual_MEComboInfoFile_False_Load_g_pas_True_Correc_iter_loadlast/num_0/holding_per_gid.hdf5" # noqa
+    }
```

Comment:
@WeinaJi : Just a comment: at a minimum, we should move this to proj12? 

In general, as we want to move the CI out of gitlab / BB5 very soon, it would be good to have tests that are not dependent on BB5/GitLab. Because very soon, we need to work on removing gitlab dependency.

Reply:
this was my comment also, would be nice to avoid depending on bb5 for the tests...

Reply:
Yes sure, for this test I will create small, local and dummy calibration files to read as I am not test the final results.